src/bin/pgcopydb/ld_apply.c

```diff
@@ -986,20 +1042,22 @@ stream_apply_sql(StreamApplyContext *context,
 			}
 
 			uint32_t hash = metadata->hash;
-			PreparedStmt *stmtHashTable = context->preparedStmt;
-			PreparedStmt *stmt = NULL;
+			bool isCopyOperation = context->currentCopyHash == hash &&
+								   context->currentCopyHash != 0;
```

Comment:
The context->currentCopyHash currently serves two purposes: is there a copy operation in flight, and which one. I think I would prefer it if we had two separate variables, one for each purpose. Maybe introduce a boolean slot `context->copyInProgress` or something?

Reply:
That makes sense, let me change the code.